### PR TITLE
Added/Modified files needed for prometheus installation

### PR DESCRIPTION
**Pull request by arshiagg**
prometheus.yml must be mounted under /var/contiv

### CODE REVIEW

File: cluster/Vagrantfile [hunk 1]
```diff
@@ -231,7 +231,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       c.vm.network :private_network, ip: node_addr, virtualbox__intnet: 'true', auto_config: false
 
       c.vm.provider 'virtualbox' do |v|
-        v.memory = node_orc == ORC_KUBEADM && master_ips.include?(node_addr) ? 2048 : 512
+        v.cpus = 2
+	v.memory = node_orc == ORC_KUBEADM && master_ips.include?(node_addr) ? 2048 : 1024
```
Comment (dseevr) [inline cluster/Vagrantfile]: Could you remove the changes to this file, amend the commit, and force push it?

**Comment by neelimamukiri** [comment]
cluster/tmp.conf & setup.sh have to be removed.

File: cluster/Vagrantfile (hunk 1 above)
Comment (arshiagg) [inline cluster/Vagrantfile]: done

File: install/k8s/k8s1.6/contiv.yaml [hunk 2]
```diff
@@ -142,6 +144,8 @@ spec:
         # Runs netplugin container on each Kubernetes node.  This
         # container programs network policy and routes on each
         # host.
+        - name: sinatra-exporter
+          image: arshiagg/sinatra
         - name: contiv-netplugin
```
Comment (rchirakk) [inline install/k8s/k8s1.6/contiv.yaml]: 1. upload image under contiv and use version, ```__CONTIV_VERSION__ ```
2. @neelimamukiri can we make this configurable ?

Comment (arshiagg) [inline install/k8s/k8s1.6/contiv.yaml]: The exporter is currently being changed to directly get stats from ovs

File: install/k8s/k8s1.6/contiv-prometheus.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  name: prometheus
+  namespace: kube-system
+spec:
+  type: NodePort 
+  selector:
+    k8s-app: contiv-prometheus 
+  ports:
+    - protocol: TCP
+      port: 9090
+      nodePort: 30000
```
Comment (neelimamukiri) [inline install/k8s/k8s1.6/contiv-prometheus.yml]: Please keep this close to the grafana nodeport.

**Pull request merged**